src/main/java/core/basesyntax/Application.java

```diff
@@ -0,0 +1,22 @@
+package core.basesyntax;
+
+public class Application {
+    private static final int arrayLenght = 6;
```

Comment:
The variable name 'arrayLenght' has a typo and should be 'arrayLength'. Also, consider following the Java naming conventions for constants which suggest using uppercase letters with underscores: 'ARRAY_LENGTH'.